internal/services/containerapps/container_app_job_resource.go

```diff
@@ -93,6 +93,7 @@ func (r ContainerAppJobResource) Arguments() map[string]*schema.Schema {
 
 		"template": helpers.JobTemplateSchema(),
 
+		// TODO rename `secrets` to `secrets` in 4.0
```

Comment:
Guessing this probably wants to be renamed to `secret`
```suggestion
		// TODO rename `secrets` to `secret` in 4.0
```